flink-metrics/flink-metrics-influxdb/src/test/java/org/apache/flink/metrics/influxdb/MeasurementInfoProviderTest.java

```diff
@@ -52,25 +49,26 @@ public void simpleTestGetMetricInfo() {
                         .build();
 
         MeasurementInfo info = provider.getMetricInfo(metricName, metricGroup);
-        assertNotNull(info);
-        assertEquals(
-                String.join("" + MeasurementInfoProvider.SCOPE_SEPARATOR, logicalScope, metricName),
-                info.getName());
-        assertThat(info.getTags(), hasEntry("A", "a"));
-        assertThat(info.getTags(), hasEntry("B", "b"));
-        assertThat(info.getTags(), hasEntry("C", "c"));
-        assertEquals(3, info.getTags().size());
+        assertThat(info).isNotNull();
+        assertThat(info.getName())
+                .isEqualTo(
+                        String.join(
+                                "" + MeasurementInfoProvider.SCOPE_SEPARATOR,
+                                logicalScope,
+                                metricName));
+        assertThat(info.getTags())
+                .containsExactly(entry("A", "a"), entry("B", "b"), entry("C", "c"));
```

Comment:
```suggestion
                .containsOnly(entry("A", "a"), entry("B", "b"), entry("C", "c"));
```
`containsExactly` requires the order to be the same, which is coincidentally the case here.